packages/mrm/bin/mrm.js

```diff
@@ -65,14 +65,15 @@ async function main() {
 		binaryPath && binaryPath.endsWith('/npx') ? 'npx mrm' : 'mrm';
 
 	// Preset
-	const preset = argv.preset || 'default';
+	const userConfig = await getConfig(defaultDirectories, 'config.json', argv);
```

Comment:
Calling `getConfig` twice looks very confusing here. Do we need to do this? If yes, could you add a comment, explaining why?

Reply:
Yeah, I agree it is a bit confusing. The issue is that the second call uses `directories` instead of `defaultDirectories` and `directories` is resolved by the call to `resolveDirectories` which is a nested function using the `preset` variable from this scope.

Maybe it would be better to call `getConfigFromFile(directories, "config.json")`?

Reply:
I guess it would be a bit more clear. Some comments won't hurt though ;-)

Reply:
Added a comment and switched to `getConfigFromFile(..)`